requirements.txt

```diff
@@ -0,0 +1,12 @@
+blinker==1.6.2
```

Comment:
https://stackoverflow.com/questions/55052434/does-python-requirements-file-have-to-specify-version